packages/ui-components/src/components/text-field/text-field.base.scss

```diff
@@ -38,6 +40,8 @@
 
 	--border-color-error: #{kv-color('error')};
 	--right-slot-width: 36px;
```

Comment:
Shoul this also be padding? $spacing-9x?

Reply:
It should but i was afraid it would introduce breaking changes. But since its a small fix, it might be changed as well

Reply:
@djcouto found this prop being used for a padding-right and a width property. I will divide both into proper naming (width and padding) and will look for breaking changes while adding this to kelvin maps